scripts/userstyles.yml

```diff
@@ -755,6 +755,16 @@ userstyles:
     readme:
       app-link: "https://vercel.com"
       current-maintainers: [*ryanccn]
+  vikunja:
+    name: Vikunja
+    categories: [productivity, note_taking]
+    color: blue
+    readme:
+      app-link: "https://vikunja.io"
+      current-maintainers: [*Guaxinim5573]
+      usage: |+
+        > [!NOTE]
+        > To add your instance, go to the userstyle's **Settings > Custom included sites** and add your instance to the list, for example, `*://vikunja.example.org/*`.
```

Comment:
This is being moved to the main readme since several themes have this same note, see https://github.com/catppuccin/userstyles/pull/724